certbot/cli.py

```diff
@@ -500,16 +499,23 @@ def prescan_for_flag(self, flag, possible_arguments):
             pass
         return True
 
-    def add(self, topic, *args, **kwargs):
+    def add(self, topics, *args, **kwargs):
```

Comment:
nit: My understanding is if `--help all` is used and multiple topics are specified, the flag will show up in the first listed topic. Perhaps this should be documented?